server/src/modules/db/schemas/videos.js

```diff
@@ -1,6 +1,7 @@
 const { baseSchema, ensureCollection } = require('./common');
 
 const VIDEO_VISIBILITIES = ['Public', 'Private', 'Unlisted'];
+const VIDEO_STATUS = ["pending", "processed", "published"];
```

Comment:
If these constants are used in multiple places, we should move these to the `constants.js` file